src/main/java/se/sundsvall/seabloader/service/InvoiceService.java

```diff
@@ -1,23 +1,37 @@
 package se.sundsvall.seabloader.service;
 
-import static java.util.Objects.nonNull;
-import static se.sundsvall.seabloader.service.mapper.InvoiceMapper.toInvoiceEntity;
-
+import org.apache.commons.collections4.ListUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
-
 import se.sundsvall.seabloader.integration.db.InvoiceRepository;
+import se.sundsvall.seabloader.integration.db.model.enums.Status;
+import se.sundsvall.seabloader.integration.invoicecache.InvoiceCacheClient;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.util.Objects.nonNull;
+import static se.sundsvall.seabloader.integration.db.model.enums.Status.FAILED;
+import static se.sundsvall.seabloader.integration.db.model.enums.Status.PROCESSED;
+import static se.sundsvall.seabloader.integration.db.model.enums.Status.UNPROCESSED;
+import static se.sundsvall.seabloader.service.mapper.InvoiceMapper.toInExchangeInvoice;
+import static se.sundsvall.seabloader.service.mapper.InvoiceMapper.toInvoiceEntity;
+import static se.sundsvall.seabloader.service.mapper.InvoiceMapper.toInvoicePdfRequest;
 
 @Service
 public class InvoiceService {
 
 	private static final Logger LOGGER = LoggerFactory.getLogger(InvoiceService.class);
+	private static final Status[] STATUSES_TO_SEND = {UNPROCESSED, FAILED};
```

Comment:
Rename to STATUSES_OF_INVOICES_TO_SEND (or something like that)?

(we are not sending statuses)